extra/src/autogluon/extra/model_zoo/_nets.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import mxnet as mx
+from mxnet import gluon, init
+
+from .model_zoo import get_model
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['get_network', 'get_built_in_network']
+
+
+# TODO: Consider deleting this file, it is only used in a single tutorial and shouldn't be a part the codebase
```

Comment:
agreed, this could just be defined in the tutorial if it's not used anywhere else

Reply:
could affect the nas functionalities, though I agree current structure is not the best way to do it

Reply:
@zhreshold if you can find time, it would be nice if you could rework the tutorial that uses this so that this code can be deleted. A lot of it appears to be largely outdated.

Reply:
Also, we should consider if we want to keep the NAS functionality or not, it isn't well maintained by us at present, and I'm concerned that it will confuse users and developers in future. I see a fair amount of old code that is only used in a few places in NAS code, and could be deleted if NAS wasn't present or if NAS was refactored.